packages/frontend/pages/vault/[vid].tsx

```diff
@@ -242,10 +242,10 @@ const SelectLP = React.memo<{ lpToken: number; setLpToken: (t: number) => void }
 
   const { data } = useQuery<positions, positionsVariables>(POSITIONS_QUERY, {
     variables: {
-      poolAddress: squeethPool?.toLowerCase(),
-      owner: address?.toLowerCase() || '',
+      poolAddress: squeethPool,
+      owner: address || '',
     },
-    fetchPolicy: 'no-cache',
+    fetchPolicy: 'cache-and-network',
```

Comment:
LP positions were not showing because of this before. Did you test that flow? Apparently is some bug with the apollo client

Reply:
ummm I tested on ropsten and it worked for me

Reply:
But can change to cache-and-network for fetchPolicy

Reply:
<img width="981" alt="image" src="https://user-images.githubusercontent.com/26922518/156655591-892fe261-0de0-4b6e-ac18-1077ffb90ca1.png">
It showed up without refreshing